config/initializers/engine.rb

```diff
@@ -0,0 +1,7 @@
+module Hyrax
```

Comment:
I think we want to find a way not override the Hyrax::Engine, but to just override the `config.to_prepare` block.